docs/compass/02-01-compass.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Basic architecture
+type: Architecture
+---
+
+Compass is a central place which stores applications and Runtimes configurations, and then propagates that information accordingly. It also plays a crucial role in establishing a trusted connection between applications and Runtimes. The basic workflow looks as follows:
```

Comment:
```suggestion
Compass is a central place which stores application and Runtime configuration and then propagates that information accordingly. It also plays a crucial role in establishing a trusted connection between applications and Runtimes. The basic workflow looks as follows:
```

Reply:
`propagates that information accordingly` maybe we can explain it a bit or at least add where it propagates that info?